spec/models/person_spec.rb

```diff
@@ -29,11 +29,11 @@
       end
 
       it 'selects distinct people' do
-        aspect = bob.aspects.create(:name => 'hilarious people')
+        aspect = bob.aspects.create(name: 'hilarious people')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.